periodic/periodic.go

```diff
@@ -106,49 +106,49 @@ func NewAborter() *Aborter {
 // RunnerOptions are the parameters to the PeriodicRunner.
 type RunnerOptions struct {
 	// Type of run (to be copied into results)
-	RunType string
+	RunType string `json:"-"`
```

Comment:
that doesn't seem right? 

also why I know it's nice to spell out the json name, the defaults do work and I tend to prefer less code, less typos over more (ie not change this file at all unless it improves something?)

Reply:
From my understanding, it must have declarations if we Marshall it like we do now. Otherwise at least we get slapped by the linter.

The other thing is that I've removed a few from the output. Vars that are not set or chans (which can't be marshalled). I'm fine of leaving the default names, but it should have some Jason declaration.

Reply:
I have a set of *Result objects too for emitting the json that fortio saves for instance - which don't have stuff that is not needed and can be a base object (but then again this isn't the result :) ) 

Maybe the shared Options are enough

but why removing RunType for instance anyway?

Reply:
From my understanding (but I will check again) it is net set. Only after it's done.

Reply:
You're right and it turns into an interesting challenge in #611